assets/tests/hooks/useSocket.test.ts

```diff
@@ -11,6 +11,7 @@ jest.mock("phoenix", () => ({
     connect: jest.fn(),
     onOpen: jest.fn(),
     onClose: jest.fn(),
+    disconnect: jest.fn(),
```

Comment:
Does this need to be here? Is there a test that relies on this that hasn't made it into the PR yet?

Reply:
This is left over from when I was experimenting with the actual cleanup function - I had the cleanup call `disconnect`, so any test involving this hook would attempt to call the `disconnect` method on unmount.